cmd/server-main.go

```diff
@@ -461,6 +464,35 @@ func initConfigSubsystem(ctx context.Context, newObject ObjectLayer) error {
 	return nil
 }
 
+func getGlobalInternodeInterface(interfs ...string) string {
+	globalInternodeInterfaceOnce.Do(func() {
+		if len(interfs) != 0 && strings.TrimSpace(interfs[0]) != "" {
+			globalInternodeInterface = interfs[0]
+			return
+		}
+		ip := "127.0.0.1"
+		host, _ := mustSplitHostPort(globalMinioAddr)
+		if host != "" {
+			ip = host
+		}
+
+		ifs, err := net.Interfaces()
+		logger.FatalIf(err, "Unable to get IP addresses of this host")
```

Comment:
Is this important enough to crash the server?

Also, crash message seems wrong. You are fetching interfaces, not IP addresses - but perhaps it shouldn't be there.